src/components/Images/Images.module.scss

```diff
@@ -20,3 +21,14 @@ figure {
   width: 100%;
   height: auto;
 }
+
+.blur-up {
+  -webkit-filter: blur(5px);
+  filter: blur(5px);
```

Comment:
Isn't autoprefixer gonna do this for us?

![image](https://user-images.githubusercontent.com/142403/68173311-73089980-ff2f-11e9-9b83-bf516e87bf0c.png)

Reply:
Fixed